app/locales/it/config.js

```diff
@@ -0,0 +1,16 @@
+// Ember-I18n includes configuration for common locales. Most users
+// can safely delete this file. Use it if you need to override behavior
+// for a locale or define behavior for a locale that Ember-I18n
+// doesn't know about.
```

Comment:
It appears that ember-i18n comes with default settings for italian, so this file can be removed. The config file is only needed for languages that we do not already know how to handle.
https://github.com/jamesarosen/ember-i18n/tree/master/addon/config